pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -43,6 +44,8 @@ import (
 
 const ResourceResyncTime = 60 * time.Second
 
+const OrphanFinalizerID = "orphanFinalizer"
```

Comment:
Can we keep core finalizers like this in the API similar to how we handle resource names?  That way we can ensure custom finalizers are qualified names in validation?